### ISSUE TITLE
Does not compile with LLVM SVN

### ISSUE THREAD

**Issue by ismail** [csmith-project/creduce#69]
I know supporting ever-changing LLVM API is not easy but this would be nice to check for known crashes with newer llvm versions. Here is the build error:

```
/havana/t/c/creduce/clang_delta/TransformationManager.cpp: In member function ‘bool TransformationManager::initializeCompilerInstance(std::__cxx11::string&)’:
/havana/t/c/creduce/clang_delta/TransformationManager.cpp:150:23: error: ‘class clang::Builtin::Context’ has no member named ‘InitializeBuiltins’
   PP.getBuiltinInfo().InitializeBuiltins(PP.getIdentifierTable(),
                       ^
```

Fix is using lower-case "initializeBuiltins" for llvm >= 3.8.0

**Comment by regehr** [comment]
ismail, could you please send a pull request to our llvm-svn-compatible branch? that is where we try to track the latest LLVM.

**Comment by ismail** [comment]
@eeide was faster :) https://github.com/csmith-project/creduce/commit/0df87ff60d5c5df528feaeae16d5ad9c2741eeb5

Thanks!

**Comment by eeide** [comment]
As noted, I fixed this issue last night, in commit 0df87ff6. Thank you for reporting it, @ismail!
